source/_integrations/unifiprotect.markdown

```diff
@@ -79,71 +83,54 @@ The Integration uses the RTSP(S) Streams as the Live Feed source, so this needs
-  * **Microphone Level** - If your camera has a microphone, a configuration number entity will be added to adjust the camera's microphone sensitivity.
-  * **WDR Level** - If your camera does not have HDR, a configuration number entity will be added to adjust the WDR level.
-* **Switch** - The following switch entities will be added for each camera:
-  * **Overlay (Show Name, Show Date, Show Logo, Show Bitrate)** - 4 configuration switches will be added to configuration the Overlay Information on your camera's video feed.
-  * **Smart Detections (Person, Vehicle)** - If your camera has smart detections, there will be a switch added for each smart detection type to enable or disable that type.
-  * **Status Light** - If your camera has a status light, there will be a configuration switch to turn it on and off.
-  * **HDR Mode** - If your camera has HDR, there will be a configuration switch to turn it on and off.
-  * **High FPS** - If your camera has a "High FPS" mode, there will be a configuration switch to toggle between Default and High FPS mode.
-  * **Privacy Mode** - If your camera allows for Privacy Masks, there will be a configuration switch to toggle a "Privacy Mode" that disables recording, microphone, and a black privacy zone over the whole camera.
-  * **System Sounds** - If your camera has a speaker, there will be a configuration switch to toggle system sounds.
-  * **SSH Enabled** - A disabled by default switch entity will be added to let you toggle on and off SSH for your camera. Username is `ubnt` and password is your Device Password as configured in UniFi Protect.
-* **Button** - A disabled by default button entity is added for each camera device. The button will let you reboot your camera device.
+* **Camera** - A camera for each camera channel and RTSP(S) combination found for each camera (up to 6). Only the highest resolution RTSPS camera entity will be enabled by default.
+* **Media Player** - If your camera has a speaker, you will get a media player entity that allows you to play audio to your camera's speaker. Any audio file URI that is playable by FFmpeg will be able to be played to your speaker, including via the [TTS Say Service](/integrations/tts/#service-say).
+* **Privacy Mode** - If your camera allows for Privacy Masks, there will be a configuration switch to toggle a "Privacy Mode" that disables recording, microphone, and a black privacy zone over the whole camera.
+* **Device Configuration** - Cameras will get various different configuration controls based on the features available to the camera. Currently provided configuration controls:
+  * configuration sliders for Chime Duration, Zoom Level, Microphone Sensitivity, and WDR Level
+  * configuration switches Overlay Information, Smart Detections types, Status Light, HDR, High FPS mode, System Sounds.
+* **Button** - A disabled by default button is added for each camera device. The button will let you reboot your camera device.
```

Comment:
@frenck How does this look? I refactor it out to not lay out every individual entity, but instead highlight important controls/sensors as well as an overview of the provided configuration entities.

Button is at the bottom for all of them and separate because it is a "control" (so important), but still disabled by default. 

Also, explicitly laid out a "UniFi Protect device" vs a HA entity so the terminology should hopefully be less confusing.